backend/app/store/service/service.go

```diff
@@ -983,7 +983,7 @@ func (s *DataStore) prepVotes(c store.Comment, user store.User) store.Comment {
 // get secret for given siteID
 // Note: secret shared across sites, but some sites can be disabled.
 func (s *DataStore) getSecret(siteID string) (secret string, err error) {
-	if secret, err = s.AdminStore.Key("any"); err != nil {
+	if secret, err = s.AdminStore.Key(siteID); err != nil {
```

Comment:
would be nice to add a comment like "note: siteID ignored for the default admin.Static store". The current comment is misleading as after this change secret is not shared (well, for some admin stores) anymore

Reply:
I've altered the comment as you proposed, but I haven't looked into the issue deeply to know the precise results of that change. I verified that it doesn't seem to be a breaking change and settled on that.

Reply:
i have checked what it can affect quickly, and it looks safe to me